lms/djangoapps/courseware/date_summary.py

```diff
@@ -143,7 +149,8 @@ class TodaysDate(DateSummary):
 
     @property
     def date_format(self):
-        return u'%b %d, %Y (%H:%M {utc})'.format(utc=_('UTC'))
+        kwargs = {'abbr': True}
```

Comment:
No need to construct this separately.  Just use `get_formatted_time_zone(self.time_zone, abbr=True)